src/DataTable/__tests__/DataTable.test.js

```diff
@@ -457,6 +457,22 @@ describe('DataTable::expandableRows', () => {
 
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('should render correctly when defaultExpandedField is true', () => {
+    const mock = dataMock();
+    mock.data[0].defaultExpanded = true;
+    const { container } = render(
+      <DataTable
+        data={mock.data}
+        columns={mock.columns}
+        defaultSortField="some.name"
```

Comment:
minor issue: I would remove this from the test since you are not setting a sortable column.

Reply:
You're totally right, I just removed it from the test code